src/packages/solid/components/Column/Column.tsx

```diff
@@ -0,0 +1,78 @@
+import { Component, createEffect, onMount, on } from 'solid-js';
```

Comment:
I think Column can just be alias to https://github.com/lightning-js/solid-primitives/blob/main/packages/Column.tsx

Or we should discuss combining them but I don't think the default column needs to do anything special

Reply:
So we don't need to build out a Column component or story? We are just going to import Column from primitives when we need to use it in other components?

Reply:
the LUI Column handles the position of the scrolling a little differently - the focused content remains centered in the screen and the other column items scroll up through it. is this something we'd want to pull into the primitive?